CHANGELOG.md

```diff
@@ -12,11 +12,19 @@ the code was deployed.
 ## [Unreleased]
 
 ### Added
-
 - Added migration script to update device_type_enum to handle multistall and singlestall sensors separately (CU-86duvwbq2).
-
-### Changed
-
+- Added `newButton.mst` mustache file for adding new buttons (CU-86dug9udj).
+- Added function `renderNewButtonPage` in `dashboard.js` to render the new mustache file (CU-86dug9udj).
+- Added validator `validateNewButton` in `dashboard.js` to validate inputs from the form (CU-86dug9udj).
+- Added function `submitNewButton` in `dashboard.js` to handle functionality of submitting new buttons (CU-86dug9udj).
+- Added function `createButtonFromBrowserForm` in `db.js` to handle inserting new buttons to the database (CU-86dug9udj).
+- Added new route for POST function of new button (CU-86dug9udj).
+- Added function `getButtonWithDeviceId` in `db.js` for integration testing of add buttons (CU-86dug9udj).
+- Added `submitNewButtonTest.js` file wiht integartion tests for adding new buttons (CU-86dug9udj).
+
+### Changed
+- Changed header to include `New Button` button (CU-86dug9udj).
+- Moved all the post functions up to put it above API routes in `routes.js` (CU-86dug9udj).
```

Comment:
Maybe you can combine changes into groups like "added dashboard functions, validateNewButton and submitNewButton in dashboard.js for adding new buttons" instead of listing them individually? 

Also add spaces after ## headers.

Reply:
fixed. i tried putting them all in one line but it gets too long